src/make.ts

```diff
@@ -359,7 +359,9 @@ export class Make {
      */
     protected async handleResponse<T>(response: Response): Promise<T> {
         const contentType = response.headers.get('content-type');
-        const result = contentType?.includes('application/json') ? await response.json() : await response.text();
+        const isJsonType: boolean = Boolean(contentType === 'application/json' || contentType?.startsWith('application/json;')); //prevent application/jsonc to be parsed as json
```

Comment:
nit: The `Boolean()` is redundant given the inner expression already evaluates to a boolean value, I would recommend removing it.

```suggestion
        const isJsonType: boolean = contentType === 'application/json' || contentType?.startsWith('application/json;'); //prevent application/jsonc to be parsed as json
```

Reply:
When the contentType is null, the isJsonType became undefined. So bcs of ts I need to cast it ;)

Reply:
oh, true, good point 👍🏻